docs/proposed/config-split.md

```diff
@@ -0,0 +1,467 @@
+Configuration file split
```

Comment:
Please add this file to `docs/proposed/Makefile`.

(Unfortunately pandoc isn't 100% compatible with GitHub: it has slightly different rules for section anchor names, even with `-f gfm`, at least as of version 2.9.2.1. But it's close, including most section anchors, tables, etc.)